extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -176,11 +194,28 @@ export class AmpIframe extends AMP.BaseElement {
   }
 
 
+  /**
+   * Relayouts are needed after the initial render if we don't have an iframe,
+   * since it may be removed when the doc becomes inactive.
+   * @override
+   **/
+  isRelayoutNeeded() {
```

Comment:
Last ditch attempt, please change `resources-impl.js Resource.documentBecameInactive` where you will find `this.state_ = NOT_LAID_OUT`. Add the following statement here: `this.layoutCount_ = 0;`.

If it works - great. If not, let's keep `isRelayoutNeeded`, file a bug and reference it here with TODO.